R-package/NAMESPACE

```diff
@@ -9,6 +9,7 @@ S3method(print,lgb.Booster)
 S3method(set_field,lgb.Dataset)
 S3method(slice,lgb.Dataset)
 S3method(summary,lgb.Booster)
+export(getLGBMthreads)
```

Comment:
I chose not to add similar functions to the Python interface, since:

* it's the R package that needs this more urgently
* taking back parts of the public API for the Python package is MUCH harder than for the R package, as the Python package is used so much more widely
* this PR is already bigger than I'm comfortable with